src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -115,7 +119,38 @@ registerAction2(class extends Action2 {
 		if (!instance) {
 			return;
 		}
-		AccessibleBufferContribution.get(instance)?.show();
+		await AccessibleBufferContribution.get(instance)?.show();
+	}
+});
+
+registerAction2(class extends Action2 {
+	constructor() {
+		super({
+			id: TerminalCommandId.GoToCommand,
+			title: { value: localize('workbench.action.terminal.goToCommand', 'Go to Command'), original: 'Go to Command' },
+			f1: true,
+			category,
+			precondition: ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
+			keybinding: [
+				{
+					primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyO,
+					weight: KeybindingWeight.WorkbenchContrib + 2,
```

Comment:
this feels really nice - when in accessibility mode, if the terminal is focused and I use cmd+shift+o, it focuses the accessible buffer and opens this go to command quick pick. 

regardless of accessibility mode, now when the accessible buffer is focused, this keybinding will go to command as well

Reply:
cmd+shift+o was established as the keybinding for opening the detected links though, if you do this then links are no longer easily accessible?

Reply:
That still prevails when not in screen reader mode. I think the discoverability of this feature is more important because it enables them to traverse the buffer